src/Umbraco.Cms.ManagementApi/DependencyInjection/RepositoryBuilderExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Umbraco.Cms.Core.DependencyInjection;
+using Umbraco.Cms.Core.Mapping;
+using Umbraco.Cms.ManagementApi.Mapping.TrackedReferences;
+using Umbraco.Cms.ManagementApi.Services;
+using Umbraco.New.Cms.Core.Persistence.Repositories;
+using Umbraco.New.Cms.Infrastructure.Persistence.Mappers;
+using Umbraco.New.Cms.Infrastructure.Persistence.Repositories.Implement;
+
+namespace Umbraco.Cms.ManagementApi.DependencyInjection;
+
+public static class RepositoryBuilderExtensions
+{
+    internal static IUmbracoBuilder AddRepositories(this IUmbracoBuilder builder)
+    {
+        builder.Services.AddScoped<ITrackedReferencesSkipTakeRepository, TrackedReferencesSkipTakeRepository>();
+        builder.Services.AddTransient<ITrackedReferencesSkipTakeService, TrackedReferencesSkipTakeService>();
+
+        builder.WithCollectionBuilder<MapDefinitionCollectionBuilder>()
+            .Add<TrackedReferenceViewModelsMapDefinition>()
+            .Add<RelationModelMapDefinition>();
```

Comment:
Isn't these already registered in `MappingBuilderExtensions`? Maybe a bad merge?